### ISSUE TITLE
Escape / Enter / Backspace not working

### ISSUE THREAD

**Issue by magicvodi** [flathub/io.mrarm.mcpelauncher#59]
Hello,
I'm on arch linux, kernel 5.14.8-zen1-1-zen, plasma desktop on X.
I have io.mrarm.mcpelauncher v0.2.6
```
$ LANG=C flatpak list
Name                                 Application ID                               Version          Branch          Installation
Minecraft Bedrock Launcher           io.mrarm.mcpelauncher                        v0.2.6           stable          system
Mesa                                 org.freedesktop.Platform.GL.default          21.1.7           20.08           system
openh264                             org.freedesktop.Platform.openh264            2.1.0            2.0             system
Breeze Gtk theme                     org.gtk.Gtk3theme.Breeze                                      3.22            system
KDE Application Platform             org.kde.Platform                                              5.15            system
```
and Minecraft 1.17.30.04 (X86_64).

It runs smooth but the Escape, Enter and Backspace Keys are not working.

Do you need more information? Any Ideas what I could try?

**Comment by ChristopherHX** [comment]
Which keyboard layout are you using?

de_DE.utf8 has working ESC / Enter / Backspace.

**Comment by magicvodi** [comment]
In KDE/Plasma I'm using german keyboard layout.
But I noticed Minecraft is using an english keyboard layout.

**Comment by ChristopherHX** [comment]
> Minecraft is using an english keyboard layout

Yes the mcpelauncher-client maps the keys to an english keyboard for input, imagine you are using a keyboard with russian letters which aren't mapable to https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes (I know that is for windows, but that applies also to Minecraft Bedrock)

I never had any problems with a german keyboard layout (intel 105 keys), all keys you mentioned are working for me.

Go ahead and give the AppImage a try https://github.com/ChristopherHX/linux-packaging-scripts/releases/tag/v0.3.0-rc1-673, it uses a different window / keyboard provider.
- Download the .AppImage file matching your system type
- chmod +x the .AppImage file
- run the file use double click or ./file.Appmage from Terminal

**Comment by marcelovbcfilho** [comment]
Well, I'm having the same issue in Fedora 35 and I use a default en_US keyboard layout...
There is some kind of log that I cant get to help to reproduce the error ?

OS: Fedora 35
